specs/Deployment/DeploymentAPI.md

```diff
@@ -143,18 +161,16 @@ by using DeploymentInitializeOptions object and setting ForceDeployment option b
 this API. This option when set will shut down the application(s) associated with WinAppSDK packages,
 update the WinAppSDK packages and restart the application(s).
 
-// OPENISSUE: If this option is set when updating main package, then whether the out of process com
-server from the main package such as push Notifications needs to be explicitly restarted needs to be
-understood. If restart it is needed, then the API will handle explicitly restarting it. This is
-under investigation.
+If this option is set when updating Singleton package, then the out of process com server from the
+Singleton package such as push Notifications is explicitly restarted.
 
-When the API is updating framework package and ForceDeployment option is set, then all dependent
+When the API is updating Framework package and ForceDeployment option is set, then all dependent
 packages that are NOT currently in use will be immediately re-installed to refer to the updated
-framework package and all dependent packages that are currently in use will be re-installed, after
-they shut down, to refer to the updated framework package.
+Framework package and all dependent packages that are currently in use will be re-installed, after
+they shut down, to refer to the updated Framework package.
```

Comment:
> after they shut down

Does this mean it will force the apps currently using the package to shut down, or does this mean that the Initialize() call could block indefinitely?

Reply:
It will only force the Main, Singleton and DDLM packages to shut down in order to update Main, Singleton and DDLM packages (ex: PushNotifications would be shutdown as it's part of Singleton package) and the shutdown packages are restarted automatically by the OS platform after their updates are complete (and in cases such as PushNotifictions which would need explicit reboot as it's a com server, we do reboot it after the update in Initialize()/Repair()).

It doesn't shutdown the WinAppSDK framework package dependent packaged apps and installs the higher version of the framework package side by side without uninstalling the older version. It does update the WinAppSDK framework package dependent packaged apps (in their package graph) to point to higher version of the framework package, once they are shutdown by the user. Once all references to older version of the framework package are updated to refer to the higher version of it, the older version is then completely uninstalled by the OS platform.

Reply:
What if an app is currently using Main/Singleton/DDLM package (and I set Force)? What happens to the app?